core/src/test/java/io/aiven/klaw/service/TopicSyncControllerServiceTest.java

```diff
@@ -177,6 +188,198 @@ public void getSyncTopics() throws Exception {
     assertThat(topicRequests).hasSize(2);
   }
 
+  @Test
+  @Order(5)
+  public void approveTopicRequestAllTopicsWhereOneTopicIsAlreadyCreated() throws KlawException {
+    // mock
+    stubUserInfo();
+    MockMultipleTopics();
+    mockGetTopicsFromEnv();
+    when(clusterApiService.approveTopicRequests(
+            any(), anyString(), anyInt(), anyString(), anyString(), any(), eq(TENANT_ID)))
+        .thenReturn(createAPIResponse(HttpStatus.OK, ApiResultStatus.SUCCESS.value))
+        .thenReturn(
+            createAPIResponse(
+                HttpStatus.OK,
+                "org.apache.kafka.common.errors.TopicExistsException: Topic 'testtopic' already exists."));
+
+    // execute
+    ApiResponse retval =
+        topicSyncControllerService.updateSyncBackTopics(
+            createSyncBackTopic(ALL_TOPICS, new String[0]));
+
+    // verfiy only 1 topic Request is sent to the manage database as the other already exists.
+    verify(handleDbRequests).requestForTopic(topicRequestCaptor.capture());
+    verify(handleDbRequests).updateTopicRequest(updateTopicRequestCaptor.capture(), eq(USERNAME));
+
+    List<TopicRequest> req = topicRequestCaptor.getAllValues();
+    List<TopicRequest> update = updateTopicRequestCaptor.getAllValues();
+    // we have two topics but one already exists so only one should be added to the database.
+    assertEquals(1, req.size());
+    assertEquals(1, update.size());
+    // we expect all request to also be updated to approved
+    assertEquals(req.size(), update.size());
+
+    verifyCaptureContents(req, update, 0, 1, "UnitTest-01");
+
+    // This should pass when clusterApi is updated
+    assertNotEquals(
+        "Error :Could not approve topic request. Please contact Administrator.",
+        retval.getResult());
+    assertEquals("success", retval.getResult());
```

Comment:
```suggestion
    assertThat(retval.getResult()).isEqualTo(ApiResultStatus.SUCCESS.value);
```

We have this assertThat from assertj core method all over.

Reply:
Have changed all assertEquals and assertNotEquals to assertThat(result).isEqualTo(expected) && assertThat(result)isNotEqualTo(expected).